<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ILMcontrol</class>
 <widget class="QWidget" name="ILMcontrol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>377</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>231</width>
    <height>242</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ILM Control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="combosetProbingRate_chan1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Slow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fast</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>He Probing rate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Measuring Interval</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Defines the time to wait [s] 
between two runs of 
all performed queries</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="spin_threadinterval">
           <property name="maximum">
            <double>3600.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupReadings">
     <property name="title">
      <string>Readings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="1">
       <widget class="QLCDNumber" name="lcdCurrentchan2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QProgressBar" name="progressLevelN2">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_9">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Nitrogen</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_8">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Helium</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_chan2_level">
        <property name="text">
         <string>Nitrogen Level</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_chan1_level">
        <property name="text">
         <string>Helium Level</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QProgressBar" name="progressLevelHe">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_chan2_current">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Nitrogen wire current</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Channel 2 cryogen</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLCDNumber" name="lcdCurrentchan1">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Channel 1 cryogen</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_chan1_current">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Helium wire current</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
